api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -71,38 +72,38 @@
  * @see org.openmrs.api.context.Context
  */
 public class ServiceContext implements ApplicationContextAware {
-	
+
 	private static final Logger log = LoggerFactory.getLogger(ServiceContext.class);
```

Comment:
Can you discard the lots of formatting in this class?

Reply:
Okay.